public/src/pages/views/UserView.tsx

```diff
@@ -99,43 +99,43 @@ export class UserView extends React.Component<IUserViewerProps, IUserViewerState
         return selector;
     }
 
-    private renderActions(user: IUserRelation): JSX.Element[] | JSX.Element {
+    private renderActions(enrol: Enrollment): JSX.Element[] | JSX.Element {
         const actionButtons: JSX.Element[] = [];
-        const tempActions = this.getAllLinks(user);
+        const tempActions = this.getAllLinks(enrol);
         if (tempActions.length > 0) {
             switch (this.props.linkType) {
                 case ActionType.Menu:
-                    return this.renderDropdownMenu(user, tempActions);
+                    return this.renderDropdownMenu(enrol, tempActions);
                 case ActionType.InRow:
-                    actionButtons.push(...this.renderActionRow(user, tempActions));
+                    actionButtons.push(...this.renderActionRow(enrol, tempActions));
```

Comment:
Why do we have `break` below here?

Reply:
I guess it is because of the switch statement. Not sure if we really need it here though.